vue/components/ui/molecules/select/select.vue

```diff
@@ -163,6 +163,10 @@ export const Select = {
             type: Array,
             default: () => []
         },
+        resetValueOnOptionsChange: {

```

Comment:
that is a big name, perhaps it would be better to reduce it to something like `resetValue`, and jsdocs above it explaining what it does